src/component.cpp

```diff
@@ -31,6 +32,7 @@ namespace libcellml {
  */
 struct Component::ComponentImpl
 {
+    std::string mMath = "";
```

Comment:
I think a `std::string` default initialises to empty so you don't need the assignment?